### PR TITLE
Toggle Button Shifts between views

### LINKED ISSUE #53: TASK - Toggle button between grid cards and table views on main page
Acceptance criteria: Should have a two part toggle button, placed according to search bar ui design document. The button should toggle between a grid card view of the table and a regular table view. When one option is selected, the other should be grayed out.

### PR DESCRIPTION
**Pull request by NathanaelPaulus**
Removed some card stuff that isn't being used yet.
Toggle button shoots over  to the right, and I am not sure right. Maybe someone else can figure it out. But the task is done.

### CODE REVIEW

File: frontend/src/components/MunchCard.tsx
Comment (kelseyp) [inline frontend/src/components/MunchCard.tsx:1]: It doesn't look like this object is being used by any other files. Should it be a part of this pull request?

File: frontend/src/components/MunchGrid.tsx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+                  <Typography variant="body2" color="text.secondary">
+                    {card.description}
+                  </Typography>
+
+                </CardContent>
+              </Card>
+              );
+            })}
+            </Box>
+        </Paper>
```
Comment (kelseyp) [inline frontend/src/components/MunchGrid.tsx]: Is card dummy data a part of this task/PR or should this section be removed until later tasking is worked? 
```suggestion
        <Paper sx={{display:props.show}} >
            <Box sx={{ flexGrow: 1}}>
            </Box>
        </Paper>
```

File: frontend/src/components/MunchGrid.tsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+  const stabilizedThis = array.map((el, index) => [el, index] as [T, number]);
+  stabilizedThis.sort((a, b) => {
+    const order = comparator(a[0], b[0]);
+    if (order !== 0) {
+      return order;
+    }
+    return a[1] - b[1];
+  });
+  return stabilizedThis.map((el) => el[0]);
+}
```
Comment (kelseyp) [inline frontend/src/components/MunchGrid.tsx]: Were these copied and pasted from another file? If they are the same verbatim, we should probably extract them into a utils.tsx file where they can be accessed by all the files.

**Review by kelseyp — CHANGES_REQUESTED** [review]

File: frontend/src/components/MunchGrid.tsx (hunk 1 above)
Comment (NathanaelPaulus) [reply frontend/src/components/MunchGrid.tsx]: removed. the data is what is displayed on the table, by the way. but I removed it in this branch and added it to a clone for later.

File: frontend/src/components/MunchGrid.tsx (hunk 2 above)
Comment (NathanaelPaulus) [reply frontend/src/components/MunchGrid.tsx]: yeah I was wanting to eventually utilize it to sort the cards the same way we do with the table, but it'll work exactly the same so moving the functionality to another space might be a good idea

File: frontend/src/components/MunchCard.tsx
Comment (NathanaelPaulus) [reply frontend/src/components/MunchCard.tsx:1]: removed. added to a clone branch for later.

**Pull request closed without merging**